backend/main.py

```diff
@@ -1,2 +1,41 @@
-print("Hello world!")
-print("Hello world!")
\ No newline at end of file
+from fastapi import FastAPI
+
+from models import Todo
+
+from datetime import datetime
+
+
+app = FastAPI()
+
+
+def create_todo():
```

Comment:
I would suggest adding the defaults as parameters.

```python
def create_todo(
    *,
    title: str = "A Todod Title",
    color: str = "red",
    tasks: list[Task] | None = None,
):
    if tasks is None:
        tasks = [] 

    todo = Todo(...)
    return todo
)
```

Important to note that we are not using an empty list in the tasks parameter due to its mutability. In this [post](works https://web.archive.org/web/20200221224620id_/http://effbot.org/zone/default-values.htm) you will find an extensive explanation